src/SSW.SophieBot/SSWSophieBot/language-generation/en-us/greeting.en-us.lg

```diff
@@ -0,0 +1,57 @@
+# GreetingHeroCard()
+- ```
+{
+    "text": "${SayHi()}",
+    "images": [
+        ${if(!user.welcomed, GreetingImage(), '')}
+    ],
+    "buttons": [
+        ${GreetingButtons()}
+    ]
+}
+```
+
+# SayHi()
+- IF: ${turn.signedInUser.givenName != null}
+    - ```
+    <b><font size=\"4\">Welcome to SSW SophieBot!</font></b><br/><br/>
+    <b>Hi ${turn.signedInUser.givenName}</b> 🖐<br/><br/>
+    ${SummaryText()}
+    ```
+- ELSE: 
+    - ```
+    <b><font size=\"4\">Welcome to SSW SophieBot!</font></b><br/><br/>
+    <b>Hi there</b> 🖐<br/>
+    ${SummaryText()}
+    ```
+
+# GreetingImage()
+- ```
+{
+    "url": "https://www.ssw.com.au/ssw/logo/sophiebot/Images/SophieBot_preview.png"
```

Comment:
We'd better not use the image URL since it might be invalid. 
- [ ] Convert it as a more high-resolution image
- [ ] Embed it as base:64 format